tests/e2e/logging/test_openshift-logging.py

```diff
@@ -72,44 +72,82 @@ def finalizer():
         helpers.wait_for_resource_state(resource=pod_obj, state=constants.STATUS_RUNNING)
         return pod_obj, pvc_obj
 
-    @retry(ModuleNotFoundError, tries=5, delay=200, backoff=1)
-    def validate_project_exists(self, pvc_obj):
+    @retry(ModuleNotFoundError, tries=5, delay=400, backoff=1)
+    def validate_project_exists(self, project):
```

Comment:
I see you changed all the usages of this function to send project instead of pvc. +1